plat/kvm/include/kvm/irq.h

```diff
@@ -36,6 +36,5 @@
 #include <sys/types.h>
 #include <uk/plat/irq.h>
 
-void _ukplat_irq_handle(unsigned long irq);
```

Comment:
Although, moving the definition to `plat/common/irq.h` is not 100% clean as Xen and linuxu don't implement this function, I think this is ok as other hardware-virtualized platforms are likely to implement it.

Since this header is basically dead after removing the definition, we should delete it entirely. It is used only in `plat/kvm/irq.c:34`, where we can also remove the reference 🧹